### PR TITLE
docs: remove legacy unused docs

### PR DESCRIPTION
**Pull request by markov00**
## Summary

This PR removes the docs, created years ago and never published.
It also removes the storybook docs addon that was used for these docs.

### CODE REVIEW

**Comment by stratoula** [comment]
I love PRs that remove code 😍

**Comment by markov00** [comment]
> I love PRs that remove code 😍

hehe, this actually removes docs but those are legacy docs :D

File: storybook/main.js [hunk 1]
```diff
@@ -10,7 +10,6 @@ module.exports = {
   stories: ['./stories/**/*.(stories|story).ts?(x)'],
   addons: [
     '@storybook/addon-actions',
-    '@storybook/addon-docs',
     '@storybook/addon-knobs',
```
Comment (nickofthyme) [inline storybook/main.js:13]: This needs to stay, the `docs` addon is required for the `storysource` to work.

PS: @stratoula another reason I loath storybook, took me DAYS to figure this out back in the day!

File: storybook/parameters.ts [hunk 2]
```diff
@@ -24,9 +24,6 @@ export const storybookParameters: Parameters = {
     selectedPanel: process.env.NODE_ENV === 'development' ? SB_KNOBS_PANEL : SB_SOURCE_PANEL,
   },
   previewTabs: {
-    'storybook/docs/panel': {
-      hidden: true,
-    },
```
Comment (nickofthyme) [inline storybook/parameters.ts:29]: I think we put this back too

**Review by nickofthyme — CHANGES_REQUESTED** [review]
Everything looks good regarding the `/docs` removal. But the `docs` addon needs to stay if we still want the story source tab to work.

File: storybook/main.js (hunk 1 above)
Comment (stratoula) [reply storybook/main.js:13]: I feel your pain Nick :D

Comment (markov00) [reply storybook/main.js:13]: oh really??? come on storysource, you can handle that yourself.... you don't need everything else

**Review by nickofthyme — APPROVED** [review]

**Pull request merged**